torch/nn/modules/loss.py

```diff
@@ -1215,7 +1221,8 @@ class TripletMarginLoss(_Loss):
 
     Shape:
         - Input: :math:`(N, D)` where :math:`D` is the vector dimension.
-        - Output: scalar. If :attr:`reduction` is ``'none'``, then :math:`(N)`.
+        - Output: If :attr:`reduction` is ``'none'``, then a tensor of shape :math:`(N)`,
```

Comment:
nit: The phrasing here is a bit weird. Maybe:
```
- Output: A Tensor of shape :math:`(N)` if :attr:`reduction` is ``'none'``, a scalar otherwise.
```